docs/examples/fa-examples.md

```diff
@@ -24,76 +39,113 @@ my_dfa = DFA(
     final_states={'q1'}
 )
 
-try:
-    while True:
-        if my_dfa.accepts_input(input('Please enter your input: ')):
-            print('Accepted')
-        else:
-            print('Rejected')
-except KeyboardInterrupt:
-    print('')
+read_user_input(my_dfa)
+```
+
+We can also do the same with an NFA we define. Note that the
+transition dictionary for the NFA has a different structure than
+that of the DFA, and that we are working over a different input
+alphabet than the previous example:
```

Comment:
@eliotwrobson Where is the machine of the previous example defined? I think the user will assume the previous example is the REPL example above, which does not show the definition of `my_automaton` (and it doesn't necessarily need to, but I think this paragraph should be reworded to be clearer / more accurate.